docs/gds-supported-platform/updating-EKS.md

```diff
@@ -0,0 +1,11 @@
+# Updating EKS
+
+* Update EKS in sandbox
+* Update nodes in sandbox
+
+If no issues are witnessed with sandbox, promote the EKS update to verify/portfolio.
```

Comment:
Does portfolio cluster exist?

Reply:
hmmm, I'm not sure, found 3 references to it in gds aws --help, so including it in case @adityapahuja

Reply:
I deleted portfolio account few months ago. I don't think it exists.

Reply:
I think it takes 3 months for an account to get deleted, but the cluster should be gone. I also notice we still have portfolio-deployer in Big Concourse though and we should tidy away that.
For this PR though: Let's just remove references to the portfolio cluster, it's not relevant any more.

Reply:
```suggestion
If no issues are witnessed with sandbox, promote the EKS update to verify.
```